UI/src/view/AssetForm.js

```diff
@@ -466,12 +466,27 @@
         dateutil.removeDatePickersFromDom();
       };
 
+      var renderLinktoWorkList = function renderLinktoWorkList() {
+        var notRendered = !$('#asset-work-list-link').length;
+        if(notRendered) {
+          $('#information-content').append('' +
+            '<div class="form form-horizontal">' +
+            '<a id="asset-work-list-link" class="floating-stops" href="#work-list/massTransitStop">Geometrian ulkopuolelle jääneet pysäkit</a>' +
+            '</div>');
+        }
+      };
+
       eventbus.on('asset:modified', function(){
         renderAssetForm();
       });
 
-      eventbus.on('layer:selected', function(layer) {
-        activeLayer = layer;
```

Comment:
`activeLayer` assignment is removed. Does this brake something? If not then we can remove `activeLayer` variable completely since it can never be anything else but `massTransitStop`